src/Oceananigans.jl

```diff
@@ -149,9 +149,22 @@ const fully_supported_float_types = (Float32, Float64)
 
 mutable struct Defaults
     FloatType :: DataType
+    gravitational_acceleration :: Float64
+    planet_angular_speed :: Float64
+end
+
+function Defaults(;
+    FloatType = Float64,
+    # [m s⁻²] conventional standard value for Earth's gravity https://en.wikipedia.org/wiki/Gravitational_acceleration#Gravity_model_for_Earth
+    gravitational_acceleration = 9.80665,
+    # [m] Earth's radius; see https://en.wikipedia.org/wiki/Earth%27s_radius
+    planet_radius = 6.371e6,
+    # [s⁻¹] Earth's angular speed; see https://en.wikipedia.org/wiki/Earth%27s_rotation#Angular_speed
+    planet_rotation_rate = 7.292115e-5)       
```

Comment:
why call it "planet" which is ambiguous?

```suggestion
    Earth_rotation_rate = 7.292115e-5)       
```

Reply:
true; agreed! ignore my suggestion, I take it back